runtime/queries/agda/highlights.scm

```diff
@@ -62,20 +71,16 @@
   "infixl"
   "infixr"
   "record"
-]
-@keyword
+] @keyword
 
 ;;;(expr 
 ;;;	f_name: (atom) @function)
 ;; Brackets
-
 [
   "("
   ")"
   "{"
-  "}"]
-@punctuation.bracket
+  "}"
+] @punctuation.bracket
 
-[
-  "="
-] @operator
+[ "=" ] @operator
```

Comment:
Nitpick: Should I be removing the braces here, or keep it as this style. The implementation in nvim-ts-format does support removing 1-sized list like this.

Reply:
(Just finished washing the dishes. Going to sleep nwo)

Reply:
I'm fine with either; keeping makes it slightly easier to add, but removing looks better.

(I'm assuming this is specific to anonymous nodes?)

Reply:
Going to keep it as it is right now. It's simpler and we can always look at this again at a later date.